### DISCUSSION TITLE
Remove key requirement for "/skyblock/news"

### DISCUSSION THREAD

**Discussion by TheNullicorn** [HypixelDev/PublicAPI#524] [Ideas]
Considering how rarely the endpoint updates, is there any reason it's not a resource endpoint? It doesn't seem like something that needs to constantly be updated. Maybe move it to `/resources/skyblock/news` and make `/skyblock/news` an alias?

Not that I use it personally, I was just looking through the docs and thought it was odd.

**Comment by lunaynx** [comment]
Bumping this, I agree. It's heavily cacheable and contains only public information, I see no reason for it to require an API key.
